src/Draco.Editor.Web/Program.cs

```diff
@@ -105,79 +65,72 @@ public static IEnumerable<string> GetAllReferencedAssemblyNames(Assembly assembl
         return processedAssemblyNames.Where(s => s != assembly).Select(s => $"{s.GetName().Name}.dll")!;
     }
 
+
     private static void RunScript(Compilation compilation)
     {
         try
         {
-            var stream = GetAssemblyStream(compilation);
-            if (stream is null) return;
-            var buffer = stream.ToArray();
+            var dllStream = new MemoryStream();
+            var irStream = new MemoryStream();
+            var emitResult = compilation.Emit(dllStream, irStream);
+            dllStream.Position = 0;
+            irStream.Position = 0;
+            var hasIR = irStream.Length > 0;
+            var hasIL = dllStream.Length > 0;
+
+            if (hasIR)
+            {
+                var text = new StreamReader(irStream).ReadToEnd();
+                SetOutputText("IR", text);
+            }
+
+            if (hasIL)
+            {
+                var text = new PlainTextOutput();
+                var disassembler = new ReflectionDisassembler(text, default);
+                using (var pe = new PEFile("_", dllStream))
```

Comment:
Why not use `using var ...` without blocks here? Less indentation overall

Reply:
This want through 4-5 PR review already, but ok :')